app/views/users/profile.html.erb

```diff
@@ -195,20 +195,39 @@
   <hr />
 
   <p>
-    <a class="btn btn-default btn-block" href="/feed/<%= @user.name %>.rss"><i class="fa fa-rss" style="color:orange;"></i> <%= t('users.profile.rss_feed_for') %> <%= @user.name %></a> 
+    <a class="btn btn-default btn-block" href="/feed/<%= @user.name %>.rss"><i class="fa fa-rss" style="color:orange;"></i> <%= t('users.profile.rss_feed_for') %> <%= @user.name %></a>
     <% if current_user && current_user.uid == @user.uid %>
-      <a href="/profile/<%= @user.name %>/edit" class="btn btn-default btn-block"><i class="fa fa-pencil"></i> <%= t('users.profile.edit_profile') %></a> 
+      <a href="/profile/<%= @user.name %>/edit" class="btn btn-default btn-block"><i class="fa fa-pencil"></i> <%= t('users.profile.edit_profile') %></a>
     <% end %>
   </p>
 
   <% if @user && current_user && @user == current_user %>
-    <button id="btn-token" class="btn btn-default btn-block">My Access Token</button>
-    <button data-toggle="modal" data-target="#socialModal" class="btn btn-default btn-block">Link Social media</button>
```

Comment:
Please maintain the order of button like this only and shift the modal code below both the buttons.